src/Controllers/CopyPatrol.php

```diff
@@ -290,6 +300,45 @@ protected function getFilterTypes() {
 	}
 
 	/**
+	 * Get cached user watchlist or re-fetch from wiki page and update redis
+	 *
+	 * @return array Usernames
+	 */
+	private function userWhitelist() {
```

Comment:
"userWhiteList" sounds like a member variable, not a function. Can we change this to getUserWhiteList() just to avoid any confusion? To quote https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP:
"Verb phrases are preferred: use getReturnText() instead of returnText()."